pkgs/development/python-modules/cppyy/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  cmake,
+  buildPythonPackage,
+  setuptools,
+  wheel,
+  pip,
+  cppyy-cling,
+  cppyy-backend,
+  CPyCppyy,
+}:
+buildPythonPackage rec {
+  pname = "cppyy";
+  version = "3.5.0";
+
+  src = fetchFromGitHub {
+    owner = "wlav";
+    repo = pname;
```

Comment:
```suggestion
    repo = "cppyy";
```

Reply:
I'll do this, but I guess the only thing it protects against is a change of the pname?